prov/shm/src/smr_util.c

```diff
@@ -306,6 +314,7 @@ int smr_create(const struct fi_provider *prov, struct smr_map *map,
 		smr_peer_addr_init(&smr_peer_data(*smr)[i].addr);
 		smr_peer_data(*smr)[i].sar_status = 0;
 		smr_peer_data(*smr)[i].name_sent = 0;
+		smr_peer_data(*smr)[i].xpmem.cap = SMR_VMA_CAP_OFF;
```

Comment:
init to NA?

Reply:
I don't want to use NA as it has no benefity. The cap is either OFF/ON. Not sure what NA would mean.